README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
+## Direct Induction of Issues by the ARG
 
-A list of active Study Groups can be found [here](https://arg.adaic.org/community-input).
+Members of the ARG monitor and comment on issues posted to this Github repository. ARG members strive to comment on all issues, but sometimes issues get missed. 
 
+The ARG may, from time-to-time, decide to directly induct a community issue into an AI without requiring the creation of a Language Study Group. In particular, the ARG may be interested in issues that identify obvious errors/flaws/bugs in the existing Ada standard. At other times, the ARG or its members may identify an issue that has particular merit but that has not attracted sufficient discussion or interest from the community to lead to a Language Study Group proposal.
 
----
-# How can I start?
+If the ARG later finds that such an AI needs significant work, the ARG might vote that the AI requires the creation of a Language Study Group to proceed. At that point the original owner of the original issue is encouraged to lead that effort.
 
-The best way to get started is to browse open issues to get a sense of how issues are structured, and to see if you find any existing issues that interest you. Consider speaking with supporters within that discussion about proposing a Language Study Group to take the issue further.
+## Language Study Groups
 
-If there is an issue that you have not found discussed, please be welcome to open a new issue. Try to give it a concise title, a succinct description, and an appropriate tag.
+Language Study Groups are created to transparently capture the work towards the generation and ongoing development of an AI to eventually be deliberated and voted on by the ARG.
 
-# Ada Interpretations (AIs)
+Language Study Groups can be created through two mechanisms. The first is by ARG vote to require a Language Study Group be created to further develop an existing AI. The second is by a direct proposal to form a Language Study Group, submitted to the ARG by an individual or group of the larger Ada community. 
 
-The ARG performs most of its work in terms of Ada Interpretations (AIs), which are documents that describe an identified issue with the Ada Reference Manual, or which propose enhancements to the Ada programming language.  Such issues may be identified within the ARG itself, or may be identified by the wider Ada community.  Historically the ada-comment mailing list has been used by members of the Ada community to identify problems or suggest enhancements.  
+Language Study Groups proposed by the community must be approved by the ARG. Once approved, those proposing the creation of the study group can work towards the creation of a new AI for formal consideration by the ARG. Language Study Group proposals from the community should be for significant changes or new features that will require rigorous design and prototyping work. 
```

Comment:
"... creation of a new AI for ..." ==> "... creation of one or more new AIs for ..."

Reply:
Fixed.